src/style.css

```diff
@@ -0,0 +1,145 @@
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.card {
+  width: 200px;
```

Comment:
do not set width directly to BEM block use mixes instead

Reply:
done